client/src/utils/helpers/notificationModalHelper.ts

```diff
@@ -0,0 +1,49 @@
+import { toTitleCase } from '~/utils/helpers/stringHelpers'
+import { INotification } from '~/utils/interfaces'
+import { STATUS_OPTIONS } from '~/utils/constants/notificationFilter'
+
+type NotificationData = { Type?: string }
+type WithData = { data?: unknown }
+
+export const getNotificationTitle = (notification: INotification): string => {
+  const { type = '', name = '', status, specificType } = notification
+  const lowerType = String(type).toLowerCase()
+  const typeParts = lowerType.split('_')
+
+  const isResolved = lowerType.includes('_resolved')
+
+  if (isResolved) {
+    let notificationData: NotificationData = {}
+
+    try {
+      const n = notification as INotification & WithData
+      const raw: unknown = typeof n.data === 'string' ? JSON.parse(n.data) : n.data ?? {}
+
+      if (raw !== null && typeof raw === 'object') {
+        notificationData = raw as NotificationData
+      }
+    } catch {
+      notificationData = {}
+    }
+
+    const notificationType = notificationData.Type?.toLowerCase()
+
+    if (notificationType === 'approve' || notificationType === 'disapprove') {
```

Comment:
```suggestion
  if (isResolved && ['approve', 'disapprove'].includes(notificationType ?? '')) {
```

let's use 'includes' instead so it's easier to read.